lib/game.rb

```diff
@@ -1,14 +1,15 @@
+# frozen_string_literal: true
+# this is the game class
```

Comment:
This comment is redundant - we can see that it's the game class

Reply:
I will delete these, they were to try and get the rubocop errors to go away.